.eslintrc

```
@@ -9,7 +9,7 @@
     "plugin:jsonc/recommended-with-jsonc",
     "plugin:jest/recommended"
   ],
-  "ignorePatterns": ["scripts"],
+  "ignorePatterns": ["scripts", "**/test-event.mjs"],
```

Just a nitpick, feel free to ignore this!

```suggestion
  "ignorePatterns": ["scripts", "components/**/test-event.mjs"],
```

---

good idea to be more specific, added this in 😄